demo/README.md

```diff
@@ -8,24 +8,49 @@ The following will provide a Tekton pipeline that can build an example python ap
 
 The details of each build type and the steps required to support Pelorus can be found in the [tekton manifest](./tekton-demo-setup/03-build-and-deploy.yaml)
 
-Example:
+
+Instructions:
+
+* Fork the konveyor/pelorus git repository to your github organization.
+
+* Add or configure your github-secret with your Github token
```

Comment:
".. that you created [here](https://github.com/settings/tokens)" to tell user where/how to create the GitHub token

Reply:
The token creation is in the docs, I'll link it up later